Dockerfile

```diff
@@ -1,6 +1,6 @@
 ## Base image with python and entrypoint scripts ##
 ## ============================================= ##
-FROM python:3.6.8-alpine3.10 AS base
+FROM python:3.6.9-alpine3.9 AS base
```

Comment:
Sorry - we are actually going to need to use alpine 3.10 moving forward. One of our alpine packages just broke compatibility with 3.9

Reply:
Do you still need to move to `alpine3.10`? I noticed that profiles is still on 3.9.

Reply:
Last Monday I ran into a build issue where one of our build dependencies broke compatibility with 3.9. It looks like it has been resolved since then - so no need for the change. But as a note, Profiles does not depend on that particular dependency.